doc/api/modules.md

```diff
@@ -6,7 +6,11 @@
 
 <!--name=module-->
 
-In the Node.js module system, each file is treated as a separate module. For
+CommonJS modules are the classic format to package JavaScript code for reuse. If
+you are interested in writing browser-compatible code, check out the
+[ECMAScript modules][] documentation.
+
+In the CommonJS module system, each file is treated as a separate module. For
```

Comment:
This is also true in ES Modules, it’s just that files are addresses as URLs.

Reply:
Yep, note that this sentence was there before – and probably before ESM was implemented in Node.js. Are you saying I should remove it, or rephrase it, or were you simply pointing out a fact?

Reply:
I'd just say, "in node, each file is treated as a separate module" such that it applies to both module systems. This will remain true even if we add HTTPS imports, and even if the language adds some form of module blocks.